src/test/java/com/epam/tc/hw3/ex1/Exercise1Test.java

```diff
@@ -0,0 +1,64 @@
+package com.epam.tc.hw3.ex1;
+
+import com.epam.tc.hw3.AbstractChromeTest;
+import java.io.IOException;
+import java.util.List;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import pages.MainPage;
+
+import static pages.MainPage.*;
+
+public class Exercise1Test extends AbstractChromeTest {
+
+
+    private static int countOfImagesOnMainPage = 4;
+    private static List<String> textUnderImages = List
+            .of("To include good practices\nand ideas from successful\nEPAM project",
+                    "To be flexible and\ncustomizable",
+                    "To be multiplatform",
+                    "Already have good base\n(about 20 internal and\nsome external projects),\nwish to get more…");
+
+    public Exercise1Test() throws IOException {
+    }
+
+    @Test(groups = {"HW2"})
+    public void exercise1() {
+        SoftAssert softly = new SoftAssert();
+        /* Open test site by URL. */
```

Comment:
Why do you need this driver.get() again, if you've already have it in your base class? 
driver.get("https://jdi-testing.github.io/jdi-light/index.html");